core/domain/feedback_jobs_one_off_test.py

```diff
@@ -253,3 +258,102 @@ def test_that_job_returns_correct_feedback_subject(self):
         self.assertEqual(threads[3].last_updated, threads_old[3].last_updated)
         self.assertEqual(threads[4].last_updated, threads_old[4].last_updated)
         self.assertEqual(threads[5].last_updated, threads_old[5].last_updated)
+
+
+class SuggestionMigrationOneOffJobTest(test_utils.GenericTestBase):
+    """Tests for the suggestion migration one-off job."""
+
+    AUTHOR_EMAIL = 'author@example.com'
+
+    EXP_ID = 'eid1'
+    TRANSLATION_LANGUAGE_CODE = 'en'
+
+    def _run_one_off_job(self):
+        """Runs the one-off MapReduce job."""
+        job_id = (
+            feedback_jobs_one_off.SuggestionMigrationOneOffJob.create_new())
+        feedback_jobs_one_off.SuggestionMigrationOneOffJob.enqueue(
+            job_id)
+        self.assertEqual(
+            self.count_jobs_in_taskqueue(
+                taskqueue_services.QUEUE_NAME_ONE_OFF_JOBS), 1)
+        self.process_and_flush_pending_tasks()
+
+    def setUp(self):
+        super(SuggestionMigrationOneOffJobTest, self).setUp()
+
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+        self.signup(self.EDITOR_EMAIL, self.EDITOR_USERNAME)
+        self.signup(self.AUTHOR_EMAIL, 'author')
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+        self.editor_id = self.get_user_id_from_email(self.EDITOR_EMAIL)
+        self.author_id = self.get_user_id_from_email(self.AUTHOR_EMAIL)
+
+        self.editor = user_services.UserActionsInfo(self.editor_id)
+
+        # Login and create exploration and suggestions.
+        self.login(self.EDITOR_EMAIL)
+
+        # Create exploration.
+        self.save_new_valid_exploration(
+            self.EXP_ID, self.editor_id,
+            title='Exploration for suggestions',
+            category='Algebra', objective='Test a suggestion.')
+
+        exploration = exp_services.get_exploration_by_id(self.EXP_ID)
+        init_state = exploration.states[exploration.init_state_name]
+        init_interaction = init_state.interaction
+        init_interaction.default_outcome.dest = exploration.init_state_name
+        exploration.add_states(['State 1'])
+
+        self.old_content = exp_domain.SubtitledHtml(
+            'content', 'old content').to_dict()
+
+        # Create content in State A with a single audio subtitle.
+        exploration.states['State 1'].update_content(self.old_content)
+        exp_services._save_exploration(self.editor_id, exploration, '', [])  # pylint: disable=protected-access
+        rights_manager.publish_exploration(self.editor, self.EXP_ID)
+        rights_manager.assign_role_for_exploration(
+            self.editor, self.EXP_ID, self.owner_id,
+            rights_manager.ROLE_EDITOR)
+
+    def test_migration_suggestion(self):
```

Comment:
Sounds weird, generally, say "test_" + what you are testing. E.g. test_suggestion_migration or test_migrate_suggestion()

Reply:
Fixed the name. Thanks!